.travis.yml

```
@@ -1,23 +1,27 @@
+dist: xenial
 language: python
-
-env:
-  - PYTHON_VERSION=3.6
+python: 3.6
 
 before_install:
-  - wget -q http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
-  - chmod +x miniconda.sh
-  - ./miniconda.sh -b -p /home/travis/miniconda
-  - export PATH=/home/travis/miniconda/bin:$PATH
+  - wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda3.sh
+  - chmod +x miniconda3.sh
+  - ./miniconda3.sh -b -p /home/travis/miniconda3
+  - export PATH=/home/travis/miniconda3/bin:$PATH
   - conda update --yes --quiet conda
 install:
-  - conda env create --file environment.yml python=${PYTHON_VERSION}
+  - conda env create --file environment.yml python=3.6
   - source activate CLaR-env
+  - python --version
   - conda install --yes --quiet pytest pytest-cov coverage
+  - pip install pytest-env
   - pip install flake8 check-manifest
   - pip install codecov
   - python setup.py build_ext --inplace
   - pip install -e .
 script:
+  - source activate CLaR-env
+  # - python3.6 -m pytest
+  - pytest
   # - pytest -lv --cov-report term-missing celer --cov=celer --cov-config .coveragerc
   - flake8 --count celer
```

this should be:

flake8 --count clar

  - flake8 --count celer
```
```suggestion
  - flake8 --count celer
```
```suggestion
  - flake8 --count celer
```